test/integration/coverage_test.dart

```diff
@@ -37,10 +40,13 @@ Future<bool> runCoverage(String projectPath, {bool html: false}) async {
   }
 
   List args = ['run', 'dart_dev', 'coverage'];
+  if (functional) {
+    args.add('--functional');
+    args.add('--no-unit');
```

Comment:
Can remove this `--no-unit` line now that I've updated the text fixture's dart_dev config to set its unit tests suite to an empty list.